composer.json

```diff
@@ -32,7 +32,7 @@
         "php": ">=7.1.0",
         "aws/aws-sdk-php": "^3.0",
         "bbc-rmp/cloudwatch-monitoringhandler": "^1.0",
-        "bbc/programmes-pages-service": "^4.0",
+        "bbc/programmes-pages-service": "^10.0",
```

Comment:
That's a pretty major version bump

Reply:
Yeah, the version in there is about a year old. We've not had any need to update it. The only changes that I _think_ should break anything are the ones to the categories methods, which this PR removes the calls to. That said it will need some regression testing.